Taskfile.yml

```diff
@@ -56,6 +56,7 @@ tasks:
           cp $FLAGS_FILE workspaces/cli-server/.env
           cp $FLAGS_FILE workspaces/cli-scripts/.env
           cp $FLAGS_FILE workspaces/optic-engine/.env
+          cp $FLAGS_FILE workspaces/optic-engine-native/.env
```

Comment:
note for posterity: we can probably remove optic-engine and cli-scripts now, since it doesn't appear they use it anymore. optic-engine's usage moved to optic-engine-native, and cli-scripts' usage moved to cli-server/optic-engine-native as part of getting rid of our scala code

Reply:
maybe this should get logged somewhere? no one is ever going to look back at comments in a merged PR for backlog items.